server/routes/index.ts

```diff
@@ -33,8 +32,8 @@ export default function routes(services: Services): Router {
     res.redirect(config.learningPlanUrl)
   })
 
-  router.get('/plan/create/:id/**/update', [getCiagPlanByIdResolver(services.ciagService)])
-  router.post('/plan/create/:id/**/update', [getCiagPlanByIdResolver(services.ciagService)])
+  router.get('/plan/create/:id/**/update', [retrieveInduction(services.ciagService)])
```

Comment:
Use the new `retrieveInduction` router request handler (below) instead of the (now deleted) "resolver"